src/components/StyledText/StyledLink/styled.ts

```diff
@@ -0,0 +1,16 @@
+import { Link } from 'react-router-dom'
+import styled from 'styled-components'
+
+type Props = {
+	changeColor?: string
```

Comment:
Again, `color` is just fine. `changeColor` implies that it should be a function. :smile: